### PR TITLE
Githika surveyrule icon

### PR DESCRIPTION
**Pull request by gtondapu**
Added icon for survey questions in collection page when a rule exists for a question.
Added format for the tooltip of collection page. Modified tooltip style.
Fixed a quote (') in surveycardlist.js

### CODE REVIEW

**Comment by gtondapu** [comment]
https://github.com/openforis/collect-earth-online/issues/1007

File: src/js/components/SurveyCollection.js [hunk 1]
```diff
@@ -350,11 +350,22 @@ class SurveyQuestionTree extends React.Component {
                     >
                         {this.state.showAnswers ? <span>-</span> : <span>+</span>}
                     </button>
+
+                    {this.props.getRulesById(this.props.surveyNode.id).length > 0
```
Comment (sirmspencer) [inline src/js/components/SurveyCollection.js:354]: You dont need a ternary here.  Use `this.props.getRulesById(this.props.surveyNode.id).length > 0 &&`

Comment (gtondapu) [reply src/js/components/SurveyCollection.js]: Done made the change and added icon!

**Comment by sirmspencer** [comment]
- You dont need the <g> tags in the SVG. g is for group, if there is not attributes in g it is unecessary.
- Some of the longer set of rules dont display well.  Can you 1) create a list with - or bullets for questions with multiple rules and 2) use a different width for the collection page and the survey design page? It looks like the survey collection panel is narrower than the rule designer.  You may want to pull out other items like left so each location can be styled a little differently.

**Comment by sirmspencer** [comment]
image

**Pull request merged**